packages/api/cms-api/src/sentry/sentry.intercepor.ts

```diff
@@ -0,0 +1,74 @@
+import { CallHandler, ExecutionContext, Injectable, NestInterceptor } from "@nestjs/common";
+import { GqlContextType, GqlExecutionContext } from "@nestjs/graphql";
+import { Observable, tap } from "rxjs";
+
+type SentryInterceptorShouldReport = (exception: unknown) => boolean;
+
+export type SentryInterceptorOptions = {
+    shouldReport?: SentryInterceptorShouldReport;
+};
+
+type Sentry = typeof import("@sentry/node");
+
+@Injectable()
+export class SentryInterceptor implements NestInterceptor {
+    shouldReport: SentryInterceptorShouldReport;
```

Comment:
Shouldn't this be injected using dependency injection?

Reply:
Done: https://github.com/vivid-planet/comet/pull/2334/commits/1ae35f2d563100ccff18e5255be137d100439d29